feat: add core values to /about page

paulatulis opened the pull request:
Values section is now populated, but I need to update CSS. Keeping this as a draft for now. 

@lottamus any initial thoughts on the layout? Is there one block that looks more desirable than the others? 

Also want to change title colors depending on team preference... 

<img width="1000" alt="Screenshot 2019-12-17 at 5 11 42 PM" src="https://user-images.githubusercontent.com/47359669/71042242-d1a96180-20f0-11ea-812e-6606daef0c02.png">

lottamus commented:
Deploy preview for *stoplightio* ready!

Built with commit 1ea72348175fbe509f517664b6c8d81dff6a336c

https://deploy-preview-1296--stoplightio.netlify.com

ksatx commented:
Styling comments - I think the values should be Title Case, without the periods afterword. i.e. 
Be An Owner 
Build Together
Practice Mindfulness

Perhaps some kind of semi-opaque background behind the Core Values? To make them pop as their own section. They look a little flat as is.

ksatx commented:
There's a lot of dead space between the values, the mission, and the top of careers. 
Maybe make "Company Mission" its own side-set feature section, like Careers and Social Good? We could use the create-wall.jpg that's already in the images folder.

![image](https://user-images.githubusercontent.com/51092436/71197939-56fb5600-2258-11ea-98b9-54e16fa491e4.png)

lottamus commented on src/components/Value/index.tsx:
```diff
@@ -28,10 +28,10 @@ export const ValueSection = ({ values, ...sectionProps }: IValueSection) => {
         <div className="container flex flex-no-wrap justify-center md:justify-around md:px-0 sm:flex-wrap">
           {values.map((value, key) => (
-            <div className="flex flex-col w-100 sm:w-100 sm:items-center sm:text-center pr-16 sm:pr-0">
+            <div className="flex flex-col text-center w-100 sm:w-100 sm:items-center sm:text-center pr-16 sm:pr-0">
```
I suggest doing something like "mx-8" instead of a right padding. If you do right padding only, it you will end up with no padding to the left of the left most element and a bunch of padding to the right of the last element. Notice the alignment with the centered text:

**Before**

![image](https://user-images.githubusercontent.com/7423098/71368456-9df58e00-256d-11ea-97f0-9a182847cab0.png)


**After**

![image](https://user-images.githubusercontent.com/7423098/71368429-8c13eb00-256d-11ea-9376-02483ae06e6a.png)


```suggestion
            <div className="flex flex-col text-center w-100 sm:w-100 sm:items-center sm:text-center mx-8">
```

lottamus replied:
Does `w-100` do anything?

```suggestion
            <div className="flex flex-col text-center sm:items-center sm:text-center pr-16 sm:pr-0">
```

lottamus replied:
Having both `text-center` and `sm:text-center` is redundant. Just use `text-center`

```suggestion
            <div className="flex flex-col text-center w-100 sm:w-100 sm:items-center pr-16 sm:pr-0">
```

lottamus commented on netlify/pages/about.yaml:
```diff
@@ -5,6 +5,31 @@ hero:
     src: /images/team/team.jpg
   title: About Stoplight
+valueSection:
+  sectionTitle: Our Values
```
I'm indifferent on this change. Technically these aren't _all_ of our values, just our "core" ones

```suggestion
  sectionTitle: Our Core Values
```

paulatulis commented:
WIP!

CSS is a mess will return to that after I get the big things taken care of

I kept both options for mission. I think it's a good opportunity to display the team photo if we don't want it in the hero.

Pulled out featureStrip into new component AnchorLinkStrip but cannot for the life of me get the AnchorLinkStrip to display all links - something is wrong with About yaml formatting I think, but unable to get to the root of ‘undefined’. Logic not worked out yet because nothing will even render at this point :)

paulatulis commented:
Okay made some additional updates. Still not complete, but want to make sure I am directionally correct before I continue disassembling.

1. Replaced current feature strip with anchor link strip that will link to every section on the page (probably a more elegant way to do this by pulling from already present yaml file sections rather than creating a new one specifically for anchorLinks, so suggestions welcome!)
2. Removed team pic from Hero, but need to figure out what to include instead
3. Put mission section at the top (placed into its own card, which is currently called 'Mission' so that I don't confuse myself at the moment, but will eventually just be FeatureCard. :) 
4. Core values now sandwiched between mission and other sections. Still need to align text for each value, but that should be helped along by [1298](https://github.com/stoplightio/stoplight.io/pull/1298)

Please note that spacing between components is not uniform, but I'm saving that fun task for the end.

paulatulis commented:
@lottamus  made some updates and would like to sync with you.
- Removed Hero, and need some help getting the layout to make way for the header
- Moved Mission to the top
- Moved group photo to Team section

ksatx commented:
![image](https://user-images.githubusercontent.com/51092436/71742871-ae88ec80-2e28-11ea-9825-df761e5eb2d0.png)
![image](https://user-images.githubusercontent.com/51092436/71742886-b8125480-2e28-11ea-96bf-52910cde9996.png)
 Looks like something funky is happening with the CSS, but I love this layout!

paulatulis commented:
@lottamus made following updates we spoke about on Friday: 
- Get rid of ‘About Stoplight’ and icon 
- Put Hero back so we can have background image
- Keep first sentence of mission in Hero
- Add CTA Join Us 
- Rework values to make them less text-heavy and add summary to top of section 

Let me know when/if you want to go over!

ksatx commented:
![image](https://user-images.githubusercontent.com/51092436/71830759-0b74e480-306d-11ea-8115-747ccbbaca82.png)
"Be An Owner" and "Build Together" have the same description. 

The doggie gif is only looping once for me, idk if that's just me. It ends on a blurry still, if it would be better to replace with an outright still image I have tons ;)
![image](https://user-images.githubusercontent.com/51092436/71830970-86d69600-306d-11ea-8ad4-51bc8940d879.png)

paulatulis commented:
> ![image](https://user-images.githubusercontent.com/51092436/71830759-0b74e480-306d-11ea-8115-747ccbbaca82.png)
> "Be An Owner" and "Build Together" have the same description.
> 
> The doggie gif is only looping once for me, idk if that's just me. It ends on a blurry still, if it would be better to replace with an outright still image I have tons ;)
> ![image](https://user-images.githubusercontent.com/51092436/71830970-86d69600-306d-11ea-8ad4-51bc8940d879.png)

Great catch! Just updated it. 

And for the gif, we need to make the file smaller, so I was playing around with that and probably messed it up. I tried uploading to Cloudinary to compress it, but that didn't help. I'll get it back on track in the next push!

paulatulis commented:
@ksatx would you be able to send over a few still images of the pups for the careers section in case I can't compress the gif enough? Thank you!

ksatx commented:
![lokihowie](https://user-images.githubusercontent.com/51092436/71853292-80154680-30a0-11ea-803e-0b533ed4f473.jpg)
![freyahowie](https://user-images.githubusercontent.com/51092436/71853294-81467380-30a0-11ea-9318-cf25cb1524fa.jpg)

paulatulis commented:
Still WIP, but getting closer...

- Probably want someone from SEO team to look at wording of all sections to make sure everything is optimized
- Swapped out dog gif for photo
- Evened out spacing among section headers and content

Let me know what you think!

ssspear commented on src/templates/About/index.tsx:
```diff
@@ -53,52 +57,80 @@ const Member: React.FunctionComponent<IMember> = ({ image, name, role, isLast })
+
+      <Section id="features" noPaddingB>
+        <Container className="pb-16 text-center border-b">
+          <LargeCard
```
Do we want to throw this info into our yaml file?

paulatulis replied:
YES great idea incorporating now

paulatulis commented on src/components/PressSection/index.tsx:
```diff
@@ -40,7 +40,7 @@ export const PressSection: React.FunctionComponent<IPressSection> = ({ id, title
 
   return (
-    <Section id={id} {...sectionProps}>
+    <Section id={id} {...sectionProps} noPaddingB>
```
noPaddingB only gets rid of padding on the bottom.

paulatulis commented on src/components/FeatureCard/index.tsx:
```diff
@@ -0,0 +1,70 @@
+import { Image } from '../Image';
+import { Link } from '../Link';
+
+export interface IMission {
```
Deleted the entire file since it's not being utilized. I had completely forgotten about this, thank you so much for flagging!

paulatulis commented:
- Removed hard coded text from component file and placed into About yaml 
- Updated "Careers" section blurb because it was the exact same as the blurb in the Team Section 
- Made Member into its own component so we can reuse that down the road

paulatulis commented:
@lottamus your changes have been implemented and I added chips on the bottom sections to give them a little color and break them up. I think this is the final version unless there's anything else that jumps out at you.
